test/verify/check-storage-format

```diff
@@ -47,12 +41,11 @@ class TestStorage(StorageCase):
         self.content_head_action(1, "Format")
         self.dialog_wait_open()
         self.dialog_set_val("type", "xfs")
+        self.dialog_set_val("mount_point", "/foo")
```

Comment:
Could you please add a test that checks invalid mount points, i. e. empty or non-absolute paths? (In the same test case, for efficiency)

Reply:
The only invalid value is the empty string.  I added a test for that.